lib/passport-configurator.js

```diff
@@ -405,6 +406,26 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
         }
       );
       break;
+    case 'fb-token':
```

Comment:
If you take a look at line 154 it loads the AuthStrategy based on the `strategy` key